firmware/src/magnet/charger.cpp

```diff
@@ -136,7 +157,10 @@ Charger::Status Charger::runAndGetStatus()
     {
         board::runPump(50, on_time_cy, off_time_cy);
     }
-
+    if (board::getOutVoltageInVolts() >= target_output_voltage_)
+    {
+         board::syslog(" Vin min = ", supply_volatage_mV_min , " mV\r\n");
```

Comment:
This give a compiler error, variable not declared in scope, what am I doing wrong?